client/src/components/application/sections/StdHeader.vue

```diff
@@ -66,8 +66,7 @@
                 </div>
               </div>
             </v-form>
-            <div id="userid" class="ml-1 mt-auto mb-auto fv-ital"
-                             :class="wordClassificationFlag ? 'mr-5 pr-5' : ''">{{ userId }}</div>
```

Comment:
wordClassificationFlag should have been removed fully from the App in a previous PR.  it just throws an error now because it doesn't exist anywhere any more